populus/compilation/compile.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import os
+
+from .helpers import (
+    get_dir_source_paths,
+    post_process_compiled_contracts,
+    validate_compiled_contracts,
+)
+
+from .backends.solc_auto import (
+    get_compiler_backend_class_for_version
+)
+
+from populus.utils.functional import (
+    get_duplicates,
+)
+
+
+from populus.config.loading import (
+    load_global_config,
+)
+
+
+def _get_contract_key(contract_data):
+    return (
+        contract_data['source_path'],
+        contract_data['name'],
+    )
+
+
+def compile_dirs(dir_paths,
+                 global_config_path=None,
+                 import_remappings=None,
+                 compiler_version="auto"
+                 ):
```

Comment:
Formatting nitpick.  This should be move up to the tail of the previous line to stay consistent with the rest of the codebase.